components/bl-dashlet/README.md

```diff
@@ -19,17 +19,17 @@ The component allows you to add a dashlet to your UI-Builder application.
 | localStorageEnabled | Checkbox                          | `false`       | Local Storage Enabled Logic | YES          | YES        | Allows to determine whether the state (size, position, closed or open) of a dashlet is allowed to be stored in [Local Storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) or not. |
 | resizing            | Checkbox                          | `false`       | Resizing Logic              | YES          | YES        | Allows to determine whether resizing is allowed or not.                                                                                                                                                      |
 | draggable           | Checkbox                          | `false`       | Draggable Logic             | YES          | YES        | Allows to determine whether dragging is allowed or not.                                                                                                                                                      |
-| contextBlock        | JSON                              | `[]`          | Context Block Logic         | YES          | YES        | Allows to determine context block. [Codeless Examples](#examples). Signature of context block: `{type, label, content}`.                                                                                     |
+| contextMenu         | JSON                              | `[]`          | Context Menu Logic          | YES          | YES        | Allows to determine context menu items. [Codeless Examples](#examples). Signature of context menu: `{type, label, content}`.                                                                                 |
```

Comment:
As far as i know we decide to use readable name of property at api table. Isn't it?

Reply:
I know, but there is already a created ticket to improve the README and I want to do it there.